tox.ini

```diff
@@ -168,3 +168,11 @@ passenv =
 skip_install = false
 allowlist_externals =
   git
+
+[testenv:changelog-draft]
+deps =
+    towncrier
+    -r{toxinidir}/docs/requirements.txt
+commands =
+    towncrier --version
+    {envpython} -c 'import sys, subprocess; subprocess.run([sys.executable, "-m", "towncrier", "build", "--version", "main", "--draft"], stderr=subprocess.PIPE)'
```

Comment:
Is this meant to eat stderr? Why wrap it in this state? Also, I usually have posargs IIRC, that allow me to pass args for experiments.

Reply:
Right. I forgot about it. Your way is cleaner. Although, I'd probably pass `subprocess.DEVNULL` there.

Reply:
Something like this
```suggestion
    {envpython} -bb -Ic 'import sys, subprocess; subprocess.check_call([sys.executable, "-bb", "-Im", "towncrier", "build", "--version", "main", "--draft"] + sys.argv[1:], stderr=subprocess.DEVNULL)' {posargs}
```